thewalrus/_permanent.py

```diff
@@ -15,29 +15,24 @@
 Permanent Python interface
 """
 import numpy as np
-
+from numba import jit
 from ._hafnian import hafnian_repeated
-from .libwalrus import perm_complex, perm_real, perm_BBFG_real, perm_BBFG_complex
 
+# pylint: disable = C0103, R0914
 
-def perm(A, quad=True, fsum=False, method="bbfg"):
-    """Returns the permanent of a matrix using the `method` formula
 
+def perm(A, method="bbfg"):
+    """Returns the permanent of a matrix using the `method` formula
     For more direct control, you may wish to call :func:`perm_real`
     or :func:`perm_complex` directly.
-
     Args:
```

Comment:
```suggestion

    Args:
```

I think this needs a line break before it to render correctly:
https://hafnian--300.org.readthedocs.build/en/300/code/thewalrus.html#thewalrus.perm

![image](https://user-images.githubusercontent.com/2959003/141964567-7adfa526-4a65-4bce-8821-8d0064ccef8f.png)